va/win32/va_win32.c

```diff
@@ -59,7 +60,7 @@ static void LoadDriverNameFromRegistry(VADisplayContextWin32* pWin32Ctx)
 #ifndef _WIN64
     BOOL isWowProcess = false;
     if (IsWow64Process(GetCurrentProcess(), &isWowProcess) && isWowProcess)
-        wcscpy(RegistryInfo.ValueName, "VAAPIDriverNameWow");
+        wcscpy(RegistryInfo.ValueName, L"VAAPIDriverNameWow");
```

Comment:
afaik, this is already merged in https://github.com/intel/libva/pull/671/files#diff-d8b70b73203726f1dcdd2908e543118f1353b61a50515196eb294e8c63689ba9 right?

Reply:
Yup fixed. Also added the WSL bits that landed recently.